src/main/java/com/agnieszkapawska/flashcards/domain/models/User.java

```diff
@@ -31,6 +31,10 @@
     private Set<Role> roles = new HashSet<>();
     @OneToMany(mappedBy = "user")
     private Set<Flashcard> flashcards = new HashSet<>();
+    @OneToOne(mappedBy = "user")
+    private FlashcardsToLearn flashcardsToLearn;
+    @OneToOne(mappedBy = "user")
+    private FlashcardsToRepeat flashcardsToRepeat;
```

Comment:
No flashcards to refresh?
BTW in some code above you're searching in 3 repositories for storage containing flashcardId. But you use search UserRepository for user with userId (you have it in answer) and then in user object you already have those repositories.

Reply:
I decided that I don't need these fields right now